index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <head>
+        <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.2/css/all.css" integrity="sha384-*********************" crossorigin="anonymous">
+      </head>
+      
+    <title>Portfolio</title>
+</head>
+<body>
+    <header>
+        <a>MZ</a>
+        <nav>
+        <i id="menu"  class="fas fa-bars"></i> 
+        </nav>
+  </header>
+  <section >
+        <div class="item">
+            <h1> I'm Massa Zoumanigui <br> Glad to see you</h1>
+        </div>
+    <div class="item">
+        <p>I’m a software developer! I can help you build a product , feature or website Look through <br> 
+            some of my work and experience! If you like what you see and have a project you need <br>
+            coded, don’t hestiate to contact me</p>
+    </div>
+            <div class="item">
+                <h2 class="CONNECT"><a href="#">LET'S CONNECT</a> </h2>
+            </div>
+          <div class="item">
+            <ul>
+                <li> <a href="https://twitter.com/zoumaniguimass1" target="_blank"><i class="fab fa-twitter"></i></a></li>
+                <li><a href="#" target="_blank"><i class="fab fa-facebook"></i></a></li>
+                <li><a href="https://github.com/massazoum" target="_blank"><i class="fab fa-github" aria-hidden="true"></i></a></li>
+                <li><a href="www.linkedin.com/in/massa-zoumanigui-1aba4525a" target="_blank"><i class="fab fa-linkedin" aria-hidden="true"></i></a></li>
+            </ul>
+           </div>
+            
+  <section>
+
+
```

Comment:
- [ ]  As the previous reviewer said😉 take note of the `font-properties`, the colors used, the spacings and the background image, and ensure your project looks like the [design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=fbdkpd08zcETlh73-0) as it is [required](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/1_setup.md#:~:text=Your%20website%20should%20look%20exactly%20like%20the%20design%20(e.g.%20font%2C%20colors%2C%20images%2C%20text%2C%20margins)%20using%20the%20templates%20in%20Figma.), Kindly fix the correct `text color`,` background image`,`fonts(size, weight,...)`, `text line height`, `paddings`, `margins`,`correct icons`, `colors`,... 👍 


>The h1 text color is #172B4D
>The h1 font-size: 40px; and font-weight: 700;
>The p text color is #344563 and font-size: 16px;
>The h1 and p font family: 'Poppins';

- [ ] [OPTIONAL] Kindly make sure, you fixed well the background image of your section as on the   [design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=fbdkpd08zcETlh73-0)👍 

> | | Figma template | | Your project | |
> |-|---------|-|--------|-|
> | | ![header](https://user-images.githubusercontent.com/57408419/219815045-ed0e0e3f-1a76-400a-a226-a66c1d7a9c8b.jpg) | | ![heade](https://user-images.githubusercontent.com/57408419/219815097-9fd203bf-fd89-4be0-8915-edb26412eb3d.jpg) | |